cmd/notify-mailer/main.go

```diff
@@ -0,0 +1,181 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	netmail "net/mail"
```

Comment:
Lets leave this name as-is and import `boulder/mail` as `bmail`, stdlib names should take precedence.